regenerate.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from typing import List
+
+import argparse
+import os
+import subprocess
+
+
+def run(*args) -> str:
+    return subprocess.run(
+        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
+
+
+def GatherPosts(posts_dir: str) -> List[str]:
+    post_paths = run('find', posts_dir, '-name',
+                     '*.md').stdout.strip().split('\n')
+    return post_paths
+
+
+def GenerateHtml(pandoc_flags: str, post_paths: List[str]):
+    for post_path in post_paths:
+        output_path = post_path[:-2] + 'html'
```

Comment:
I think we should place output in a single root-level directory to make it easier to build the whole site.

This also doesn't handle copying figures or JS, or generating pages like the index or category lists. Maybe that's out of scope for this PR?

Reply:
> I think we should place output in a single root-level directory to make it easier to build the whole site.

Done.
> 
> This also doesn't handle copying figures or JS, or generating pages like the index or category lists. Maybe that's out of scope for this PR?

I'd prefer to do that in a separate PR.